### PR TITLE
Players turn

### PR DESCRIPTION
**Pull request by benjaminpayoux**
[Trello](https://trello.com/c/SSwXDGgz/9-05-as-jean-peter-i-would-like-to-be-able-to-play-on-a-mobile-device)

# TODO
- [X] Players can shoot at the opponent fleet
- [X] Active player switch automatically
- [X] Visually see sunk ships

# Result
last

### CODE REVIEW

File: pages/games/[hash].js [hunk 1]
```diff
@@ -1,19 +1,61 @@
...
-            <h1 className="text-2xl">Game page {data.game_hash}</h1>
-
+            <h1 className="text-2xl">Partie {data.game}</h1>
+            {isCurrentPlayer ? (
+                <h2 className="text-green-500 font-bold uppercase">
+                    A moi de jouer
+                </h2>
+            ) : (
+                <h2 className="text-orange-500 font-bold uppercase">
+                    En attent de l'adversaire
```
Comment (Luwangel) [inline pages/games/[hash].js]:
```suggestion
                    En attente de l'adversaire
```

File: pages/games/[hash].js [hunk 2]
```diff
@@ -56,13 +97,60 @@ function Game({ data }) {
...
+                                        <div
+                                            key={x + y}
+                                            className="w-full h-full"
+                                            onClick={() => shoot(x, y)}
+                                        ></div>
+                                    ))
+                                )}
+                            </div>
+                        ) : (
+                            <></>
```
Comment (Luwangel) [inline pages/games/[hash].js]: Use null instead in order to render nothing

Comment (Luwangel) [reply pages/games/[hash].js]: You can also write `{ isCurrent Player && (<div>...</div>) }`

**Review by Luwangel — CHANGES_REQUESTED** [review]
Great! It will be time (probably in another PR) to introduce components in order to clarify the code and don't repeat some logic.

File: pages/games/[hash].js (hunk 1 above)
Comment (benjaminpayoux) [reply pages/games/[hash].js]: Completely agree :+1:

**Pull request merged**